frontend/src/app/libs/auth-pages/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,34 @@
+<div>
+  <form [formGroup]="registerForm" (ngSubmit)="submit()">
+    <div>
+      <mat-form-field appearance="fill">
+        <mat-label>Enter your email</mat-label>
+        <input matInput placeholder="pat@example.com" type="email" formControlName="email" />
+        <mat-error *ngIf="(registerFormControl.email.touched || submitted) && !registerFormControl.email.valid">
+          {{ emailInputErrorMessage }}</mat-error>
```

Comment:
We can use mapped validation error messages from form `group.errors` which return array of `ValdiationError` with messages inside.

Reply:
done